pkgs/by-name/dm/dmt-core/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, buildPythonApplication, fetchurl, fetchPypi, pythonPackages }:
+
+let
+  reprint = buildPythonApplication rec {
+    pname = "reprint";
+    version = "0.6.0";
+    src = fetchurl {
```

Comment:
Prefer `fetchPypi`, and use `hash` instead of `sha256`

Reply:
I've had trouble using `fetchPypi` It's what I used earlier but lots of 404 despite following the recommendations in the documentation. Also, DMT_core is the package name as formatted in PyPi. dmt-core will fail.